app/models/hiera_data/yaml_file.rb

```diff
@@ -47,8 +47,7 @@ def content_for_key(key)
       return value if [String, Integer, Float].include?(value.class)
 
       value_string = value.to_yaml
-      value_string.gsub!("---\n", '').gsub!(/^$\n/, '')
-      value_string
+      value_string.sub(/\A---(\n| )/, '').gsub(/^$\n/, '')
```

Comment:
there is another case which i can remember seeing often at customers `--- {}`. do we handle this?

Reply:
Yes, this is more or less the exact same case as far as this code is concerned.